Changes

```diff
@@ -1,5 +1,23 @@
 Release history of Zonemaster
 
+v2018.1 2018-06-25 Public release
+
+High lights:
+
+- Zonemaster-LDNS is now built on latest version if LDNS and has been
+  modulized to make it possible to use available LDNS library.
+- Zonemaster-GUI has been rewritten from ground up.
+
+This version of Zonemaster also consists of the following components. For each
+component, see its Changes file.
+
+  Zonemaster-LDNS     v1.1.0   https://github.com/dotse/zonemaster-ldns
```

Comment:
https://github.com/dotse/zonemaster-ldns/blob/develop/Changes

Reply:
Then it should be the master branch, not the develop branch. But I think it makes more sense to point at the entire component when we say "This version of Zonemaster also consists of the following components".

Reply:
Yes, I thought i made a edit. It is below.

Reply:
I still think it makes more sense to point at the component. In the release tag in Github I will have links to component, release tag and Changes. E.g. see https://github.com/dotse/zonemaster/releases/tag/v2017.4.2